### PR TITLE
#1129 - Missing step in tutorial

### PR DESCRIPTION
**Pull request by JuanjoA**
Related to  [#1129 ](https://github.com/django/channels/issues/1129)

### CODE REVIEW

File: docs/tutorial/part_4.rst [hunk 1]
```diff
@@ -142,6 +142,22 @@ suites for end-to-end tests (``StaticLiveServerTestCase`` or ``LiveServerTestCas
 that URLs inside the Channels routing configuration like ``/ws/room/ROOM_NAME/``
 will work inside the suite.
 
+We are using ``sqlite3``, which for testing, is treated as a database in memory, and therefore, the tests will not run correctly.
```
Comment (andrewgodwin) [inline docs/tutorial/part_4.rst]: "is treated as a database in memory" -> "is run as an in-memory database"

File: docs/tutorial/part_4.rst [hunk 2]
```diff
@@ -142,6 +142,22 @@ suites for end-to-end tests (``StaticLiveServerTestCase`` or ``LiveServerTestCas
...
+We need to tell our project that the ``sqlite3`` database need not to be in memory for run the tests. Edit the
+``mysite/settings.py`` file and add ``TEST`` argument to the **DATABASES** setting::
```
Comment (andrewgodwin) [inline docs/tutorial/part_4.rst]: "add test argument" -> "add the test argument"

Comment (JuanjoA) [reply docs/tutorial/part_4.rst]: Thanks again!

**Comment by andrewgodwin** [comment]
Thanks!

**Pull request merged**